go.mod

```diff
@@ -7,7 +7,7 @@ module github.com/dell/csi-vxflexos/v2
 
 //replace github.com/dell/goscaleio => ./goscaleio
 
-//replace github.com/dell/gocsi => ./gocsi
+replace github.com/dell/gocsi => ./gocsi
 
```

Comment:
are we using local gosci , as per PR heading we have a new version to use ?

Reply:
we are using a local gocsi, yes
changes in new gocsi can be found here: https://github.com/dell/gocsi/pull/8